lib/Minz/Request.php

```diff
@@ -71,6 +71,15 @@ public static function currentRequest() {
 			'params' => self::$params,
 		);
 	}
+	public static function modifiedCurrentRequest(array $extraParams = null) {
+		$currentRequest = self::currentRequest();
+		if (is_array($extraParams)) {
+			foreach ($extraParams as $key => $value) {
```

Comment:
Wouldn't `array_merge()` (or similar array function) work better here?

Reply:
Of course! I'll change that.